manifests/g/Google/AndroidStudio/2021.2.1.15/Google.AndroidStudio.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2021.2.1.15
```

```suggestion
PackageVersion: 2021.2
```
This must match Control Panel.